Tasks/PublishTestResults/publishtestresults.ts

```diff
@@ -30,10 +30,11 @@ else {
   var matchingTestResultsFiles = [testResultsFiles];
 }
 
-if(!matchingTestResultsFiles) {
+if(!matchingTestResultsFiles || matchingTestResultsFiles.length == 0) {
```

Comment:
I think == is strongly typed and === is not strongly typed. I guess it is better to use == in typescript as it complies with language rules. Let me know your thoughts